lib/hyacinth/xml_generator/element.rb

```diff
@@ -0,0 +1,116 @@
+module Hyacinth
+  class XMLGenerator::Element
+    attr_reader :generator, :field_values
+    delegate :generate_field_val, :value_with_substitutions, :value_for_field_name,
+             :generate_field_val, :render_output_of_ternary, :render_output_of_join,
```

Comment:
```suggestion
             :render_output_of_ternary, :render_output_of_join,
```
`:generate_field_val` was in there twice.